src/main/webapp/user-page.html

```diff
@@ -150,57 +150,55 @@ <h4>About Me</h4>
       <div class="container">
         <div class="row container">
           <div class="col px-0">
-      <h4 style="margin-bottom: 20px;"> Location </h4>
+            <h4 style="margin-bottom: 20px;"> Location </h4>
             <h4>Map</h4>
             <div class="container" style="margin: 20px; width: 100%;">
-            <!-- <div id="floating-panel">
-                <input id="address" type="textbox" value="">
-                <input id="submit" type="button" value="Add Location">
-              </div> -->
-              <div id="map"></div>
-            
+       
+            <div id="map" onload="geocodeMap();">
               <script>
-                function geocodeMap() {
-                  var map = new google.maps.Map(document.getElementById('map'), {
-                    zoom: 8,
-                    center: {lat: -34.397, lng: 150.644}
-                  });
-                  var geocoder = new google.maps.Geocoder();
+              	function geocodeMap() {
+				  var map = new google.maps.Map(document.getElementById('map'), {
```

Comment:
spacing should be 2 spaces. Please fix. You can copy and paste the javascript into https://prettier.io/playground/ and have it format it for you.